src/Assignment2/Person.cs

```diff
@@ -0,0 +1,121 @@
+
+        /// <summary>
+        /// Method will get the details of the person
+        /// </summary>
+        public void GetDetails()
+        {
+            Console.Write($"\nContact ID: {this._id}\n Name: {this._name}\n Phone Number: {this._phone}\n Email Address: {this._email}\n Notes: {this._additionalNotes}\n");
+        }
+
+        /// <summary>
+        /// Method is to check and display the persons details
+        /// </summary>
+        /// <param name="element">Preference to search</param>
+        /// <returns>Boolean value to validate</returns>
+        public bool CheckAndDisplayConatctDetails(string element)
+        {
+            if (element == this._name || element == this._phone || element == this._email)
+            {
+                this.GetDetails();
+                return true;
+            }
+
+            return false;
+        }
+
+        /// <summary>
+        /// Method to set the values in contactlist
+        /// </summary>
+        /// <param name="name">Contact Nmae</param>
+        /// <param name="phone">_phone Number</param>
+        /// <param name="email">_email Address</param>
+        /// <param name="note">Additional information about the contact</param>
+        public void AddList(string name, string phone, string email, string note)
+        {
+            this._name = name;
+            this._phone = phone;
+            this._email = email;
+            this._additionalNotes = note;
+        }
+
+        /// <summary>
+        /// Method to check and edit the contact of matched id
+        /// </summary>
+        /// <param name="iD">Generated Contact _id</param>
+        /// <returns>Boolean value to validate</returns>
+        public bool IsEdit(int iD)
+        {
+            if (iD == this._id)
+            {
+                Program main = new Program();
+                Console.WriteLine("Given Details : ");
+                this.GetDetails();
+
+                Console.Write("\nEnter new Name : ");
+                this._name = Console.ReadLine();
+                Console.Write("Enter new Phone_Number : ");
+                this._phone = Console.ReadLine();
+                Console.Write("Enter new Email : ");
+                this._email = Console.ReadLine();
+                Console.Write("Enter new Notes : ");
+                this._additionalNotes = Console.ReadLine();
+            }
+            else
+            {
+                return false;
+            }
+
+            Console.WriteLine($"\nModefied Details Successfully ");
+            return true;
+        }
+
+        /// <summary>
+        /// Method to check whether the user given _id and availabble _id is same.
+        /// </summary>
+        /// <param name="id">Contact _id</param>
+        /// <returns>Boolean value to validate</returns>
+        public bool ValidateContactId(int id)
+        {
+            if (id == this._id)
+            {
+                return true;
+            }
+
+            return false;
```

Comment:
Line break can be adjusted as per suggestion

```suggestion
            return false;
```